### ISSUE TITLE
Hide cursor not working

### ISSUE THREAD

**Issue by VegeTato** [4ian/GDevelop#6028]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/4ian/GDevelop/issues)

### Describe the bug

As the title says, after the last few updates of GDevelop, the action **Hide cursor** most of the time is not working in GDevelop preview, it works on other platforms (GDgames/itch...etc), but in gdevelop preview it's not, as you can see in the video, many times it doesn't hide the cursor, and few times it does.

https://github.com/4ian/GDevelop/assets/81715237/167cc726-08f3-4193-87f5-835c07261ab3

### Steps to reproduce

1. Create new empty project.
2. Condition: at the beginning of the scene/ Action: Hide cursor.
3. Cursor most of the time won't be hidden.

### GDevelop platform

Desktop

### GDevelop version

Latest version

**Comment by ClementPasteau** [comment]
I can reproduce the issue.
It seems that it depends on the mouse being detected or not in the screen when the action is triggered.
If the mouse was outside of the window, or not detected as in the window, it will not be hidden on the frame "beginning of scene".

To confirm this, if you remove the "At the beginning of the scene", it triggers on every frame, and the cursor is always hidden.
Can you confirm, removing the condition makes it work @VegeTato ?

**Comment by VegeTato** [comment]
> I can reproduce the issue. It seems that it depends on the mouse being detected or not in the screen when the action is triggered. If the mouse was outside of the window, or not detected as in the window, it will not be hidden on the frame "beginning of scene".
> 
> To confirm this, if you remove the "At the beginning of the scene", it triggers on every frame, and the cursor is always hidden. Can you confirm, removing the condition makes it work @VegeTato ?

Yes, I can confirm it works perfectly after removing the condition At the beginning of the scene.
